### ISSUE TITLE
Auth error on login with 2 accounts with the same email

### ISSUE THREAD

**Issue by TheExGenesis** [TheExGenesis/community-archive#77]
Someone messaged me this:

> I think there might be an issue signing in with two accounts registered on the same email? Just a guess, but login stopped working for me when I tried to pivot to my other account. (Got one uploaded, thank-you for this effort!)

> https://www.community-archive.org/auth/auth-code-error#error=server_error&error_code=500&error_description=Error+getting+user+email+from+external+provider

> tbf it was fairly flaky with the first account too though so idk.
 

> Twitter page, no token found or smth.

> (I think it was an app token but sadly didn’t screenshot it mentally or otherwise).

**Comment by DefenderOfBasic** [comment]
I wonder if we want a script/workflow for "manual admin uploads" for cases like this where we want to unblock a user until the root cause is resolved (like the deleted/banned account stuff too)? Not sustainable but would be handy especially if you get someone with a big archive & they're excited to share it 

(like at least when the data is in you can start exploring your data with the ecosystem of tools/building on it. I found myself extra motivated to contribute once my data is in there because while developing I could test it on my tweets which felt very powerful)

**Comment by TheExGenesis** [comment]
yeah we do I also have a few archives of ppl who just emailed them to me

**Comment by TheExGenesis** [comment]
prob won't fix, but ppl can send me their archives and I'll upload manually
